crates/cxx-qt/src/lib.rs

```diff
@@ -76,3 +76,120 @@ pub trait Threading: Locking + Sized {
     #[doc(hidden)]
     fn threading_drop(cxx_qt_thread: &mut CxxQtThread<Self>);
 }
+
+/// This trait can be implemented inside the [`#[cxx_qt::bridge]`](bridge) on any [CxxQtType] to define a
+/// custom constructor in C++ for the QObject.
+///
+/// The `Arguments` must be a tuple of CXX types that will be the arguments to the constructor in C++.
+///
+/// Note that implementing this trait will stop Cxx-Qt from generating a default constructor.
+///
+/// # Minimal Example
+/// ```
```

Comment:
put `rust` after the ``` ?

Reply:
not needed; rustdoc presumes code blocks are Rust unless specified otherwise

Reply:
I agree with @Be-ing , `rust` is the default, so no need to specify